### ISSUE TITLE
iOS SDK version 3.6.5 -> 3.9.0

### ISSUE THREAD

**Issue by souly1** [samzilverberg/cordova-mixpanel-plugin#124]
Hey, I'm having an issue on iOS - The app gets stuck on the splash screen.
I'm receiving this exception:

```
[MPSwizzler swizzleSelector:onClass:withBlock:named:async:]: unrecognized selector sent to class
```

looking at the code it seems this would happen around the automaticPushTracking settings in the iOS SDK which was automatically set to YES.
Manually changing the value to NO solved the issue (I do not require push tracking).

in version 3.9.0 the default was set to NO, so perhaps consider upgrading?

from version 3.9.0 documentation:
```
Set automaticPushTracking and trackCrashes to NO by default when init the SDK #942
```

thanks

**Comment by samzilverberg** [comment]
published 4.7.2. with ios and android lib updates:
https://www.npmjs.com/package/cordova-plugin-mixpanel/v/4.7.2

I made this update from a non-mac computer so wasn't able to verify that everything actually works! :grimacing: 
I believe should because the changes between the mixpanel versions are pretty minimal.
Let me know if there are problems compiling with the new version.

**Comment by victorsosa** [comment]
closed issue as resolved
